### PR TITLE
README.rst - update CI example

### PR DESCRIPTION
**Pull request by keradus**
- add progress
- don't modify local files on CI
- don't care about cache as we are taking only git modified files, so we know they have been modified

### CODE REVIEW

**Comment by SpacePossum** [comment]
LGTM :+1: 
(btw. is the `-v` req. here or more recommended?)

**Comment by keradus** [comment]
none of added flags are required.
`-v` is to show better output for CI

File: README.rst [hunk 1]
```diff
@@ -892,7 +892,7 @@ Then, add the following command to your CI:
 
 .. code-block:: bash
 
-    $ vendor/bin/php-cs-fixer fix --config=.php_cs.dist --path-mode=intersection `git diff --name-only $COMMIT_RANGE`
+    $ vendor/bin/php-cs-fixer fix --config=.php_cs.dist -v --dry-run --using-cache=no --path-mode=intersection `git diff --name-only $COMMIT_RANGE`
```
Comment (keradus) [inline README.rst]: `--diff-filter=ACMRTUXB` should be added to not pass removed files

**Comment by keradus** [comment]
Travis is dead again?

**Comment by GrahamCampbell** [comment]
Just slow as shit. ;(

**Pull request closed without merging**